pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -55,6 +55,11 @@ func CmdCreate() *cobra.Command {
 	cmd.Flags().StringSlice("allowed_buckets", nil,
 		"Set the user allowed buckets list (use commas or multiple flags)")
 	cmd.Flags().String("default_resource", "", "Set the default resource, on which new buckets will be created")
+	cmd.Flags().Bool("nsfs_account_config", false, "This flag is for creating nsfs account")
```

Comment:
not sure why we need this flag? can't we understand from getting uid/gid that we want to use nsfs?

Reply:
1. We want it to be explicit, as nsfs account config is optional.
2. uid/gid have no defaults, and in an even that we want nsfs config we want to pop a prompt that ask the values.

Reply:
uid/gid have -1 default no? if we get one of these we can pop a prompt - not that I really sure why this prompt is needed - you can just fail requests with illegal/missing uid/gid.